hr_grade_rank/views/hr_grade_rank_view.xml

```diff
@@ -0,0 +1,159 @@
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
@ilakkia-serpentcs,  you can remove this line

Reply:
removed as mentioned